docs/cli/edgedb_migration.rst

```diff
@@ -24,13 +24,69 @@ description. It's also possible to split the schema definition across
 multiple ``.esdl`` files. The migration tools will read all of them
 and treat them as a single SDL document.
 
+Here is typical file tree::
+
+    ❯ tree dbschema
+    dbschema
+    ├── default.esdl
+    └── migrations
+        ├── 00001.edgeql
+        ├── 00002.edgeql
+        ├── 00003.edgeql
+        ├── ....
+        └── 00019.edgeql
+
+In the tree:
+
+* ``dbschema/*.esdl`` is a set of files that keep current database schema
+* ``default.esdl`` is named after ``module default`` that is usually used for
+  initial application schema. You don't have to stick to single file per
+  module, but this is a good start.
+* ``dbschema/migrations/*.edgeql`` is a sequence of migration files that when
+  applied bring database schema to the current. These files are mostly managed
```

Comment:
> bring database schema to the current.

bring database schema to the latest version?

Reply:
Fixed